README.md

```diff
@@ -1,2 +1,8 @@
-# kcjug.github.io
-Static site content for KCJUG
+# Attribution
+
+Site built with [Gatsby](https://www.gatsbyjs.org). From [gatsby-starter-strata](https://github.com/ChangoMan/gatsby-starter-strata).
+
+Background (header.jpg) sourced from Wikimedia Commons, from the NOAA Photo Library. Licensed under the Creative Commons [Attribution 2.0 Generic](https://creativecommons.org/licenses/by/2.0/deed.en) license.
+
+Source URL: https://commons.wikimedia.org/wiki/File:Amer0338_(19157544483).jpg
+Last accessed 04 September 2019.
```

Comment:
I think it would be valuable to provide details on how to run this locally in the README. Remember that our audience is Java developers, so it is possible they don't have node installed either.